chia/cmds/cmds_util.py

```diff
@@ -207,18 +206,18 @@ async def get_wallet(root_path: Path, wallet_client: WalletRpcClient, fingerprin
     return selected_fingerprint
 
 
-async def execute_with_wallet(
+@asynccontextmanager
+async def get_wallet_client(
     wallet_rpc_port: Optional[int],
```

Comment:
it's surprising to me to have this parameter be optional, but not default it to `None`

Reply:
fixed